packages/renderer/src/lib/preferences/Util.ts

```diff
@@ -48,7 +48,7 @@ export function writeToTerminal(xterm: any, data: string[], colorPrefix: string)
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 function writeMultilineString(xterm: any, data: string, colorPrefix: string): void {
-  if (data.includes('\n')) {
+  if (data && data.includes && data.includes('\n')) {
```

Comment:
why do you need to check `data.includes`? If it's not undefined, it will be a string and the includes function can be accessed

Reply:
it's because at runtime, it's not respecting the typescript signature so we don't receive a string object.

Reply:
probably we should use `any` instead of string